composer.json

```diff
@@ -3,20 +3,20 @@
     "description": "Deptrac is a static code analysis tool that helps to enforce rules for dependencies between software layers.",
     "license": "MIT",
     "require": {
-        "php": ">=5.5.9",
+        "php": "^5.6 || ^7.0",
         "graphp/graphviz": "0.2.*",
         "nikic/php-parser": "^3.0",
         "padraic/phar-updater": "^1.0",
         "phpdocumentor/graphviz": "~1.0",
         "sensiolabs-de/astrunner": "dev-master",
-        "symfony/config": "^2.7",
-        "symfony/console": "~2.7",
-        "symfony/dependency-injection": "^2.7",
-        "symfony/event-dispatcher": "^2.7",
-        "symfony/filesystem": "~2.7",
-        "symfony/finder": "~2.7",
-        "symfony/options-resolver": "^2.7",
-        "symfony/yaml": "^2.7"
+        "symfony/config": "^2.8 || ^3.0",
+        "symfony/console": "~2.7 || ^3.3",
```

Comment:
why sometime `^3.3` and sometimes `^3.0`?

Reply:
fixed